bin/dotd

```diff
@@ -303,6 +304,9 @@ def dotd_menu(dotd_name)
         @logger.info 'kills tests'
       end
       menu.choice(:DTP) {do_dtp(dotd_name)}
+      menu.choice(:STP) do
+        run_on 'production_daemon', 'production/bin/seed-from-staging'
+      end
```

Comment:
nitpick on naming, is 'seed from staging' more broad than what we're specifically doing?

Reply:
I think it captures the concept pretty specifically, but am open to a better phrase.

Reply:
I'd prefer `seed-curriculum-content-from-staging-branch`
It's longer but explains exactly what it's doing.